src/styles/_variables.scss

```diff
@@ -0,0 +1,20 @@
+//landing page header
+$mars-header-background: #ff9d1e;
+$mission-header-background: #ffebd1;
+$header-font: #dd361c;
+
+//base colors
+$color-base: #212121;
+$color-white: #ffffff;
+$color-primary: #105bd8;
+$color-primary-darker: #0b3d91;
+$color-primary-darkest: #061f4a;
+$color-grey-light: #aeb0b5;
```

Comment:
"light grey" is a description of the colour, not its usage. What is this colour going to be used for? If it's just a general grey that will get used across the whole site, then this is okay, but otherwise it should be named for its usage.